GCC/CORTEX_A53_64-bit_UltraScale_MPSoC/portASM.S

```diff
@@ -517,46 +517,6 @@ Exit_IRQ_No_Context_Switch:
     ERET
 
 
-
-.align 8
-
-
-/*
- *  int32_t lSpinTrylock(uintptr_t ulGateWord);
- */
-    .global lSpinTrylock
-    .type lSpinTrylock  , %function
-lSpinTrylock:
```

Comment:
Why did this function as well as `vSpinUnlock` need to move out of this file?

Reply:
Implemented in port.c using inline functions

Reply:
Implemented in port.c using inline functions